.config/matugen/templates/hyprlunavsc.json

```
@@ -0,0 +1,1934 @@
+    "editorGutter.addedBackground": "{{colors.secondary.default.hex}}80",
+    "editorGutter.deletedBackground": "{{colors.error.default.hex}}80",
+    "editorCodeLens.foreground": "{{colors.outline.default.hex}}",
+    "editorGroup.border": "{{colors.outline.default.hex}}",
+    "editorGroup.dropBackground": "{{colors.primary_container.default.hex}}40",
+    "editorGroupHeader.noTabsBackground": "{{colors.background.default.hex}}",
+    "editorGroupHeader.tabsBackground": "{{colors.surface.default.hex}}",
+    "editorGroupHeader.tabsBorder": "{{colors.outline.default.hex}}00",
+    "editorWidget.foreground": "{{colors.on_surface.default.hex}}",
+    "editorWidget.background": "{{colors.surface.default.hex}}",
+    "editorWidget.border": "{{colors.outline.default.hex}}",
+    "editorWidget.resizeBorder": "{{colors.primary.default.hex}}",
```

Biome (2.1.2)

[error] The key editorWidget.resizeBorder was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)